src/EnergyPlus/WeatherManager.cc

```diff
@@ -103,31 +103,25 @@ namespace WeatherManager {
     // weather, retrieving data from weather files, and supplying the
     // outdoor environment for each time step.
 
-    constexpr std::array<std::string_view, static_cast<int>(EpwHeaderType::Num)> epwHeaders({"LOCATION",
-                                                                                             "DESIGN CONDITIONS",
-                                                                                             "TYPICAL/EXTREME PERIODS",
-                                                                                             "GROUND TEMPERATURES",
-                                                                                             "HOLIDAYS/DAYLIGHT SAVING",
-                                                                                             "COMMENTS 1",
-                                                                                             "COMMENTS 2",
-                                                                                             "DATA PERIODS"});
+    constexpr std::array<std::string_view, (int)EpwHeaderType::Num> epwHeaders = {
```

Comment:
I'm making the executive decision to use untype-checked casts to upcast `enum` to `int`. Fight me.